src/cpu_logic.h

```diff
@@ -1,20 +1,11 @@
-#ifndef CODE_80005FD0_H
-#define CODE_80005FD0_H
+#ifndef cpu_logic_H
+#define cpu_logic_H
 
 #include "vehicles.h"
 #include "camera.h"
-#include "waypoints.h"
+#include "path.h"
 #include <assets/common_data.h>
 
-struct struct_801642D8 {
-    /* 0x0 */ u16 unk0;
-    /* 0x2 */ s16 ffff;
-    /* 0x4 */ s16 timer; // confirm?
-    /* 0x6 */ s16 laps;  // confirm?
-    /* 0x8 */ s32 blank;
-    /* 0xC */ s32 unkC;
-};
```

Comment:
Why is it that this struct is deleted. and yet replaced with `CpuItemStrategyData` which is 2 struct members bigger?
It makes sense that they might be the same, but it's weird that it matched like that. Sort of

Reply:
for no because the function only take a pointer so after he only do the offset from this pointer no matter if the struct are bigger then that